### ISSUE TITLE
Sourcemaps not working

### ISSUE THREAD

**Issue by wazzzap** [webpack-contrib/less-loader#183]
I have created simple project which demonstrates the problem:
https://github.com/wazzzap/react-tests

If i set sourceMap for `css-loader` and `less-loader` final result is empty sourcemap.
If i disable sourceMap for `less-loader` i see only css sourceMap:
https://github.com/wazzzap/react-tests/blob/master/webpack.conf.js#L53

Reproduce the problem:
```
git clone https://github.com/wazzzap/react-tests.git
cd react-tests
npm install
npm start
```
Open browser `http://localhost:9000` inspect the button. `App.less` shows empty content.

**Comment by jhnns** [comment]
Thx for that reproducible example project 👍 
Fix shipped with `4.0.3`

**Comment by wazzzap** [comment]
Np, i hope it helped. Thanks for the fix! :+1:
